atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/watch/LockWatchManager.java

```diff
@@ -37,4 +39,6 @@
      * @param lastKnownVersion exclusive start version to get events from
      */
     TransactionsLockWatchEvents getEventsForTransactions(Set<Long> startTimestamps, IdentifiedVersion lastKnownVersion);
+
+    CommitUpdate getCommitUpdate(long startTimestamp, UUID requestIdToExclude);
```

Comment:
Instead of this being a requestIdToExclude, please can this actually be a LockToken?